2-Browsers/Week1/assignment/ex4-whatsTheTime/index.js

```diff
@@ -7,7 +7,14 @@ Full description at: https://github.com/HackYourFuture/Assignments/tree/main/2-B
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  // TODO complete this function
+  const displayTime = document.createElement('p');
+  document.body.appendChild(displayTime);
+
+  setInterval(() => {
+    const currentDate = new Date().toLocaleTimeString('nl-nl');
```

Comment:
Nice and succinct solution. Will it now always show the time in the Netherlands?

Reply:
If you are in the Netherlands, of course!
Otherwise, we would need to add the timezone of the targeted country, in our situation that will be Netherlands

```suggestion
    const currentDate = new Date().toLocaleTimeString('nl-nl', {timeZone: 'Europe/Amsterdam',});
```